docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,7 +13,9 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Will return ``NULL`` for any statistics that are not populated or
```

Comment:
To be 100% correct, you might want to mention that some stats (even if the source has them) aren't supported by Trino. e.g. min/max for dates in some connectors.

Reply:
Do not use future tense